protocols/migration/migration_reports_3_0_0_to_reports_4_2_0_SNAPSHOT.py

```diff
@@ -39,40 +47,61 @@ def migrate_interpretation_request_rd(self, old_interpretation_request_rd):
         new_interpretation_request.analysisReturnUri = old_interpretation_request_rd.analysisReturnURI
         new_interpretation_request.internalStudyId = ''
 
-        if new_interpretation_request.validate(new_interpretation_request.toJsonDict()):
-            return new_interpretation_request
-        else:
-            raise Exception(
-                'This model can not be converted: ', handle_avro_errors(new_interpretation_request.validate_parts())
-            )
+        return self.validate_object(
+            object_to_validate=new_interpretation_request, object_type=self.new_model.InterpretationRequestRD
+        )
+
+    def migrate_interpreted_genome_rd(self, old_interpreted_genome_rd):
+        """
+        :type old_interpretation_request_rd: reports_3_0_0.InterpretedGenomeRD
+        :rtype: reports_4_2_0_SNAPSHOT.InterpretedGenomeRD
+        """
+        new_interpreted_genome_rd = self.new_model.InterpretedGenomeRD.fromJsonDict(
+            jsonDict=old_interpreted_genome_rd.toJsonDict()
+        )
+
+        new_interpreted_genome_rd.interpretationRequestId = old_interpreted_genome_rd.InterpretationRequestID
+        new_interpreted_genome_rd.reportUrl = old_interpreted_genome_rd.reportURL
+        new_interpreted_genome_rd.reportUri = old_interpreted_genome_rd.reportURI
+
```

Comment:
Aren't we missing here migration from VersionControl to ReportVersionControl?

Reply:
Yup, good spot @priesgo 
I've made a commit here: https://github.com/genomicsengland/GelReportModels/commit/04523f7d928f5a3d7abe64abd8eab4576b7da2e6
What do you think of the explicit migration and the extra test ?